### PR TITLE
Stating to put tests

### PR DESCRIPTION
**Pull request by eliasreis54**
#### What does this PR do?
Initial tests in actions
#### Where should the reviewer start?
the index.test.js in actions folder
#### Screenshots (if appropriate)

This PR is related to #47

### CODE REVIEW

File: src/actions/index.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  LOAD_NETWORK,
+  CHANGE_NETWORK_PROPERTY,
+  ADD_NODE,
+  newNetwork,
+  loadNetwork,
+  changeNetworkProperty,
+  addNode,
+} from './index';
+
+describe('index file tests', () => {
```
Comment (nolleto) [inline src/actions/index.test.js]: You can describe this test as `Index Actions` 😄

File: src/actions/index.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  it('newNetwork ', () => {
+    expect(newNetwork).toBeInstanceOf(Function);
```
Comment (nolleto) [inline src/actions/index.test.js]: I believe you don't need this test: checking if is a `Function` for this and all other methods.
If it changes to be not a `Function` (what it probably won happen) the other tests about this action will break 😄

File: src/actions/index.test.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    newNetwork((ret) => {
+      expect(ret.type).toEqual(NEW_NETWORK);
+      expect(ret.kind).toEqual(NETWORK_KINDS.BN);
```
Comment (nolleto) [inline src/actions/index.test.js]: Actually, this snippet of code is never triggered and so, this test will never fail.
And even if it was would be better to check the hole `ret` to make sure nothing changes, EX.:

This test (if works) would never fail if `newNetwork` added a new prop:
```js
{
  type: NEW_NETWORK,  // checked by the test
  kind: NETWORK_KINDS.BN,  // checked by the test
  newProp: 'any-value',  // NEVER checked by the test
}
```

One tip about making tests, it's a good practice always make it fail at least once just to make sure your test is testing properly. Ex.:
```js
expect(ret.type).toEqual('pão de bataaaata'); // this should fail
```
;D

**Comment by eliasreis54** [comment]
Good! Thanks to leave your review. 
I Got that suggestion on issue and change that code i made. I'll improve or add more tests after...

**Pull request merged**